### ISSUE TITLE
Pages with content AND only 1 pre tag are falsely parsed

### ISSUE THREAD

**Issue by keyosk** [acrogenesis/jsonview-safari#6]
Pages that have valid content, and additionally have only one pre tag, are falsely parsed by this extension as JSON, you should modify your code to accommodate this.

On this line https://github.com/acrogenesis/jsonview-safari/blob/master/jsonview.safariextension/jsonview.js#L4

Instead of doing:

```
if(document.getElementsByTagName("pre")[0] && document.getElementsByTagName("pre").length == 1){
```

Consider doing this:

```
if(document.querySelectorAll("*").length === 1 && document.getElementsByTagName("pre")[0] && document.getElementsByTagName("pre").length == 1){
```

**Comment by acrogenesis** [comment]
This is a known bug I've been trying to solve and I think you are in the right direction but the problem with `document.querySelectorAll("*")` is that it will also return the `html`, `head` & `body` tags too so It's the same problem again when can I know if there is content or not

**Comment by acrogenesis** [comment]
What do you think about `document.body.getElementsByTagName('*').length;`

**Comment by keyosk** [comment]
That should do nicely! :+1:

**Comment by acrogenesis** [comment]
:+1:  solved in 99429d1ee9a3e6302658cad8347a9cdca8219694
